scraper.rb

```diff
@@ -45,15 +59,29 @@ def scrape_list(url)
       phone: tds[5].xpath('.//i[@class="icon-bell"]//following-sibling::text()[contains(.,"+54")]').text.tidy,
       facebook: tds[5].css('a[href*="facebook"]/@href').text.tidy,
       twitter: tds[5].css('a[href*="twitter"]/@href').text.tidy,
-      # 6 year terms, so we should split these all into three, but for
-      # now just take the latest one
-      term: 2015,
       source: person_url,
     }
     # }.merge(scrape_person(person_url))
     data[:image] = URI.join(url, data[:image]).to_s unless data[:image].to_s.empty?
-    ScraperWiki.save_sqlite([:id, :term], data)
+    memberships << data
```

Comment:
it would probably be a little clearer / more Ruby-ish to change that `.each` to a `.map`, rather than rolling your own version of that with list concatenation.